### DISCUSSION TITLE
Running via the source files?

### DISCUSSION THREAD

**Discussion by submeg** [psoukie/zipchord#160] [Q&A] [answered]
Hi,

Is it possible to run Zipchord via the source ahk files, as opposed to the exe? Every option I have tried has been blocked or deleted (I also use AHK a lot and so prefer to work with uncompiled exes by default).

**Comment by submeg** [comment, marked as answer]
Never mind, worked it out - in the version 2 code, just run the ZipChord. ahk file.

**Comment by psoukie** [comment]
> Never mind, worked it out - in the version 2 code, just run the ZipChord. ahk file. 

Yes, correct! I am in the process of separating the various modules because I gradually learned how to approximate object oriented approach in AHK, including custom workarounds for the UI so it wouldn't flood the code with global variables. I couldn't work on this for a few weeks but will resume this week.

The idea is that the AHK files will always be able to run without relying on a compiled version.
